music21/percussion.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         percussion.py
+# Purpose:      music21 classes for representing unpitched events
+#
+# Authors:      Michael Scott Cuthbert
+#               Christopher Ariza
+#
+# Copyright:    Copyright © 2006-2019 Michael Scott Cuthbert and the music21 Project
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+
+import unittest
+
+from music21 import common
+from music21 import chord
+from music21 import note
+
+
+class PercussionChord(chord.ChordBase):
+    '''
+    :class:`~music21.chord.ChordBase` and `:class:`~music21.note.NotRest` subclass that is NOT
+    a :class:`~music21.chord.Chord` because one or more notes is an :class:`~music21.note.Unpitched`
+    object.
+
+    >>> pChord = percussion.PercussionChord([note.Unpitched('D4'), note.Note('E5')])
+    >>> pChord.isChord
+    False
+
+    Has notes, just like any ChordBase:
+
+    >>> pChord.notes
+    (<music21.note.Unpitched object at 0x...>, <music21.note.Note E>)
```

Comment:
next TODO -- improve repr for Unpitched (probably with Instrument name rather than displayStep)